pkgs/servers/akkoma/default.nix

```diff
@@ -0,0 +1,215 @@
+          sha256 = "sha256-ZYhZTe7cTITkl8DZ4z2IOlxTX5gnbJImu/lVJ2ZjR1o=";
+        };
+
+        postInstall = "mv $out/lib/erlang/lib/crypt-${version}/priv/{source,crypt}.so";
+
+        beamDeps = with final; [ elixir_make ];
+      };
+      elasticsearch = beamPackages.buildMix rec {
+        name = "elasticsearch";
+        version = "1.0.1";
+
+        src = fetchFromGitea {
+          domain = "akkoma.dev";
+          owner = "AkkomaGang";
+          repo = "elasticsearch-elixir";
+          rev = "6cd946f75f6ab9042521a009d1d32d29a90113ca";
+          hash = "sha256-CtmQHVl+VTpemne+nxbkYGcErrgCo+t3ZBPbkFSpyF0=";
+        };
+
+        beamDeps = with final; [ ];
+      };
+      gettext = beamPackages.buildMix {
+        name = "gettext";
+        version = "0.19.1";
+
+        src = fetchFromGitHub {
+          owner = "tusooa";
+          repo = "gettext";
+          rev = "72fb2496b6c5280ed911bdc3756890e7f38a4808";
+          hash = "sha256-V0qmE+LcAbVoWsJmWE4fwrduYFIZ5BzK/sGzgLY3eH0=";
+        };
+
+        beamDeps = with final; [ ];
+      };
+      linkify = beamPackages.buildMix rec {
+        name = "linkify";
+        version = "0.5.2";
+
+        src = fetchFromGitea {
+          domain = "akkoma.dev";
+          owner = "AkkomaGang";
+          repo = "linkify";
+          rev = "2567e2c1073fa371fd26fd66dfa5bc77b6919c16";
+          hash = "sha256-e3wzlbRuyw/UB5Tb7IozX/WR1T+sIBf9C/o5Thki9vg=";
+        };
+
+        beamDeps = with final; [ ];
+      };
+      mfm_parser = beamPackages.buildMix rec {
+        name = "mfm_parser";
+        version = "0.1.1";
+
+        src = fetchFromGitea {
+          domain = "akkoma.dev";
+          owner = "AkkomaGang";
+          repo = "mfm-parser";
+          rev = "912fba81152d4d572e457fd5427f9875b2bc3dbe";
+          hash = "sha256-n3WmERxKK8VM8jFIBAPS6GkbT7/zjqi3AjjWbjOdMzs=";
+        };
+
+        beamDeps = with final; [ phoenix_view temple ];
+      };
+      remote_ip = beamPackages.buildMix rec {
+        name = "remote_ip";
+        version = "0.1.5";
+
+        src = fetchFromGitLab {
+          domain = "git.pleroma.social";
+          group = "pleroma";
+          owner = "elixir-libraries";
+          repo = "remote_ip";
+          rev = "b647d0deecaa3acb140854fe4bda5b7e1dc6d1c8";
+          sha256 = "0c7vmakcxlcs3j040018i7bfd6z0yq6fjfig02g5fgakx398s0x6";
+        };
+        beamDeps = with final; [ combine plug inet_cidr ];
+      };
+      search_parser = beamPackages.buildMix rec {
+        name = "search_parser";
+        version = "0.1.0";
+
+        src = fetchFromGitHub {
+          owner = "FloatingGhost";
+          repo = "pleroma-contrib-search-parser";
+          rev = "08971a81e68686f9ac465cfb6661d51c5e4e1e7f";
+          hash = "sha256-sbo9Kcp2oT05o2GAF+IgziLPYmCkWgBfFMBCytmqg3Y=";
+        };
+
+        beamDeps = with final; [ nimble_parsec ];
+      };
+      temple = beamPackages.buildMix rec {
+        name = "temple";
+        version = "0.9.0-rc.0";
```

Comment:
```suggestion
        version = "unstable-2022-08-18";
```

Reply:
I have verified that inconsistency between the `version` attribute and the version set in `mix.lock` breaks the application at run time. But since these dependencies are not exposed as independent packages, I assume that not following the guidelines laid out in <https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming> may be acceptable.